src/components/Collections/Table/Table.vue

```diff
@@ -93,7 +93,6 @@
                   :class="tableRowClass"
                   @mouseover="tableRowHoverOn(rowIndex)"
                   @mouseout="tableRowHoverOff"
-                  @touchstart="tableRowTouchOn(rowIndex)"
```

Comment:
<img width="398" alt="Screenshot 2021-05-14 at 10 40 04 AM" src="https://user-images.githubusercontent.com/13185699/118224433-c94d8580-b4a0-11eb-8fb2-c47663420f5c.png">
breaks in 320 px screen.

Reply:
![Uploading Screenshot 2021-05-14 at 12.27.44 PM.png…]()
Looks fine on my side??

Reply:
<img width="647" alt="Screenshot 2021-05-14 at 12 29 56 PM" src="https://user-images.githubusercontent.com/13185699/118233821-18e77d80-b4b0-11eb-8149-536edd85c597.png">
staging pe bhi fatt raha

Reply:
<img width="391" alt="Screenshot 2021-05-14 at 12 45 32 PM" src="https://user-images.githubusercontent.com/20169829/118235361-47feee80-b4b2-11eb-90d7-88804adc5e78.png">
Not that bad in my case
URL is long so thoda problem hai which will be fixed in the change I've made to reduce the size